src/statemachine.rs

```diff
@@ -618,6 +618,8 @@ impl StateMachine {
                         }
                     }
 
+                    send_status(&status, crate::StatusUpdate::PresenceRequired);
```

Comment:
I think we need this also 10 lines up, where we create a dummy-credential to make the device blink.

Reply:
Good catch! I also had to add it to `legacy_register` and `legacy_sign`.